universal-application-tool-0.0.1/app/views/admin/questions/QuestionRow.java

```diff
@@ -0,0 +1,67 @@
+package views.admin.questions;
+
+import static j2html.TagCreator.a;
+import static j2html.TagCreator.span;
+import static j2html.TagCreator.td;
+
+import j2html.tags.Tag;
+import java.util.Locale;
+import services.question.QuestionDefinition;
+import services.question.TranslationNotFoundException;
+
+enum QuestionRow {
```

Comment:
praise: I like how this encapsulates the logic for rendering a table cell. One nit on naming: should the name of the enum be `QuestionTableCell`? A specific instance of the enum is for a cell rather than a row...

Reply:
Done.